src/main.rs

```diff
@@ -47,10 +47,39 @@ fn main() {
 
 fn parse_arguments<'a>(app: App<'a, '_>) -> ArgMatches<'a> {
     let app = app
+        .setting(AppSettings::TrailingVarArg)
         .version(crate_version!())
         .author(crate_authors!())
         .about("Rust implementation of the Python language")
-        .arg(Arg::with_name("script").required(false).index(1))
+        .usage("rustpython [OPTIONS] [-c CMD | -m MODULE | FILE | -] [PYARGS]...")
```

Comment:
I'd rather not advertise this usage, since it is not true, we do not support `-`.